src/lib.rs

```diff
@@ -1,26 +1,21 @@
-#![feature(external_doc)]
-#![doc(include = "../README.md")]
-#![deny(missing_docs)]
-#![cfg_attr(test, deny(warnings))]
-#![cfg_attr(test, feature(plugin))]
-#![cfg_attr(test, plugin(clippy))]
-
-extern crate console;
-extern crate failure;
-extern crate unindent;
-
-use console::style;
-use failure::Error;
+#![cfg_attr(feature = "nightly", deny(missing_docs))]
```

Comment:
This should be enabled on every channel I reckon

Reply:
I can't enable missing_docs on stable because then it'll complain about missing module docs! (As loading the Readme is only done in nightly.)

Reply:
Ahaha, yeah alright - fair enough! There's hacks we could do, but let's not do them.